src/components/common/multi-entry/multi-entry.styles.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button } from "@lifesg/react-design-system";
+import { Accordion } from "@lifesg/react-design-system/accordion";
+import styled from "styled-components";
+
+export const MultiEntryAccordionItem = styled(Accordion.Item)`
+    padding-bottom: 2rem;
+    width: 100%;
+
+    #content-container {
+        padding: 0;
+    }
+
+    [data-testid="accordion-item-expandable-container"] {
+        padding: 0 1rem;
+        display: flex;
+        flex-direction: column;
+    }
+
+    [data-testid="accordion-item-title"] {
+        margin-bottom: 1rem;
+    }
+`;
```

Comment:
layout differs from Figma mocks
- accordion title should have 1rem white space on the left
- accordion arrow should have 1rem white space on the right